src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.chopshop166.chopshoplib.commands.SmartSubsystemBase;
+import com.chopshop166.chopshoplib.motors.SmartMotorController;
+
+import java.util.function.BooleanSupplier;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.maps.RobotMap.TelescopeMap;
+
+public class Climber extends SmartSubsystemBase {
+
+  // Constants:
+  private final double EXTEND_SPEED = 1.0;
+  private final double RETRACT_SPEED = 1.0;
```

Comment:
It might be helpful to define a single direction as "forward" and make both `EXTEND_SPEED` and `RETRACT_SPEED` use that same direction for positive.

Reply:
I meant set `EXTEND_SPEED = 1.0` and `RETRACT_SPEED = -1.0`, not to introduce a new `UP`.